dosomething-dev/fastly-backend/main.tf

```diff
@@ -144,14 +144,14 @@ resource "fastly_service_v1" "backends-dev" {
 
   response_object {
     name              = "robots.txt deny"
-    content           = file("${path.root}/shared/deny-robots.txt")
+    content           = file("${path.module}/deny-robots.txt")
     request_condition = "path-robots"
   }
 
   snippet {
     name    = "Shared - Set X-Origin-Name Header"
     type    = "fetch"
-    content = file("${path.root}/shared/app_name.vcl")
+    content = file("${path.module}/app_name.vcl")
```

Comment:
Here's an example of why `app_name.vcl` now needs to be duplicated in each individual module – we can't (for security reasons, I assume) reach outside the "root" of the workspace's module to load files & so this needed to be swapped to `path.module` and moved within the `dosomething-dev/fastly-backend` directory.